front/src/modules/ui/link/components/RoundedLink.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Link as ReactLink } from 'react-router-dom';
+import styled from '@emotion/styled';
+
+import { Chip } from '@/ui/chip/components/Chip';
+import { ChipSize, ChipVariant } from '@/ui/chip/components/Chip';
+
+type OwnProps = {
+  href: string;
+  children?: React.ReactNode;
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+};
+
+const StyledClickable = styled.div`
+  overflow: hidden;
+  white-space: nowrap;
+
+  a {
+    color: inherit;
+    text-decoration: none;
+  }
+`;
+
+export function RoundedLink({ children, href, onClick }: OwnProps) {
+  return (
+    <div>
+      {children !== '' ? (
```

Comment:
then we can remove this condition

Reply:
This condition filters out links which are empty and avoids rendering them if we remove it then something like the below SS appears on empty links

![image](https://github.com/twentyhq/twenty/assets/29349136/6b3e43fb-0bec-48e6-a625-72e883ea763f)

![image](https://github.com/twentyhq/twenty/assets/29349136/895b081c-3a23-4599-a0b9-5bcf4c8c61e8)

Reply:
Got it thank you!